client-sdk/ts-web/rt/src/callformat.ts

```diff
@@ -1,4 +1,105 @@
+import * as oasis from '@oasisprotocol/client';
+
+import * as ed from '@noble/ed25519';
+import * as types from './types';
+import * as mrae from './mrae';
+
+var deoxysii = require('deoxysii');
+
 /**
  * Call data key pair ID domain separation context base.
  */
 export const CALL_DATA_KEY_PAIR_ID_CONTEXT_BASE = 'oasis-runtime-sdk/private: tx';
+
+/**
+ *  EncodeConfig is call encoding configuration.
+ */
+export interface EncodeConfig {
+    /**
+     * publicKey is an optional runtime's call data public key to use for encrypted call formats.
+     */
+    publicKey?: types.KeyManagerSignedPublicKey;
+}
+
+export interface metaEncryptedX25519DeoxysII {
```

Comment:
```suggestion
export interface MetaEncryptedX25519DeoxysII {
```

for consistency with TS naming conventions, and link to the Go src please